spec/services/validate_participant_spec.rb

```diff
@@ -118,7 +118,7 @@
             .and_return(eligible_record)
         end
 
-        it "does not save the validation data" do
+        xit "does not save the validation data" do
```

Comment:
Is that going back in when Phil's PR with new dqt api goes in? https://github.com/DFE-Digital/early-careers-framework/pull/1059

Reply:
Yes, once we can access the previous inductees we won't keep the validation data